src/actions/search.js

```diff
@@ -78,7 +83,19 @@ export function searchPostsWithParams(teamId, params, includeDeletedChannels = f
 }
 
 export function searchPosts(teamId, terms, isOrSearch, includeDeletedChannels) {
-    return searchPostsWithParams(teamId, {terms, is_or_search: isOrSearch}, includeDeletedChannels);
+    return searchPostsWithParams(teamId, {terms, is_or_search: isOrSearch, include_deleted_channels: includeDeletedChannels, page: 0, per_page: 20});
+}
+
+export function getMorePostsForSearch() {
+    return async (dispatch, getState) => {
+        const teamId = getCurrentTeamId(getState());
+        const {params, isEnd} = getState().entities.search.current[teamId];
+        if (!isEnd) {
+            const newParams = Object.assign({}, params);
```

Comment:
It's more common for us to use the spread operator `{...params}` instead of Object.assign to copy objects lately

Reply:
Is that really better? It seems less clear what is happening. (not that Object.assign is particularly clear)

Reply:
It's not particularly better, but it would be more consistent. Also you could extend that to
```
const newParams = {
    ...params,
    page: params.page + 1,
};
```
to combine it with the next line if you wanted. 0/5 though